src/BloomTests/web/EnhancedImageServerTests.cs

```diff
@@ -162,5 +170,99 @@ public void CanRetrieveContentOfFakeTempFile_ButOnlyUntilDisposed()
 				Assert.That(transactionFail.StatusCode, Is.EqualTo(404));
 			}
 		}
+
+		private void SetupCssTests()
+		{
+			// create collection directory
+			Directory.CreateDirectory(_collectionPath);
+
+			// settingsCollectionStyles.css
+			var cssFile = Path.Combine(_collectionPath, "settingsCollectionStyles.css");
+			File.WriteAllText(cssFile, @".settingsCollectionStylesCssTest{}");
+
+			// create book directory
+			var bookPath = Path.Combine(_collectionPath, "TestBook");
+			Directory.CreateDirectory(bookPath);
+
+			// languageDisplay.css
+			cssFile = Path.Combine(bookPath, "languageDisplay.css");
+			File.WriteAllText(cssFile, @".languageDisplayCssTest{}");
+
+			// Factory-XMatter.css
+			cssFile = Path.Combine(bookPath, "Factory-XMatter.css");
+			File.WriteAllText(cssFile, @".factoryXmatterCssTest{}");
+
+			// miscStyles.css - a file name not distributed with or created by Bloom
+			cssFile = Path.Combine(bookPath, "miscStyles.css");
+			File.WriteAllText(cssFile, @".miscStylesCssTest{}");
+		}
+
+		[Test]
+		public void GetCorrect_LanguageDisplayCss()
+		{
+			using (var server = CreateImageServer())
+			{
+				SetupCssTests();
+				var cssFile = Path.Combine(_folder.Path, "TestCollection", "TestBook", "languageDisplay.css");
+
+				var url = cssFile.ToLocalhost();
+				var transaction = new PretendRequestInfo(url);
+
+				server.MakeReply(transaction);
+
+				Assert.That(transaction.ReplyContents, Is.EqualTo(".languageDisplayCssTest{}"));
+			}
+		}
+
+		[Test]
+		public void GetCorrect_SettingsCollectionStylesCss()
+		{
+			using (var server = CreateImageServer())
+			{
+				SetupCssTests();
+				var cssFile = Path.Combine(_folder.Path, "TestCollection", "settingsCollectionStyles.css");
+
+				var url = cssFile.ToLocalhost();
+				var transaction = new PretendRequestInfo(url);
+
+				server.MakeReply(transaction);
+
+				Assert.That(transaction.ReplyContents, Is.EqualTo(".settingsCollectionStylesCssTest{}"));
+			}
+		}
+
+		[Test]
+		public void GetCorrect_XmatterStylesCss()
+		{
+			using (var server = CreateImageServer())
+			{
+				SetupCssTests();
+				var cssFile = Path.Combine(_folder.Path, "TestCollection", "TestBook", "Factory-XMatter.css");
+
+				var url = cssFile.ToLocalhost();
+				var transaction = new PretendRequestInfo(url);
+
+				server.MakeReply(transaction);
+
+				Assert.AreNotEqual(transaction.ReplyContents, ".factoryXmatterCssTest{}");
```

Comment:
Could we somehow assert that it found some other file? E.g., this test would pass if it responded "file not found".